xml/System.Runtime/AmbiguousImplementationException.xml

```diff
@@ -51,8 +51,8 @@
         <Parameter Name="message" Type="System.String" />
       </Parameters>
       <Docs>
-        <param name="message">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="message">The localized error message string.</param>
```

Comment:
```suggestion
        <param name="message">A <see cref="System.String"/> that describes the error.</param>
```

(Borrowed from https://docs.microsoft.com/en-us/dotnet/api/system.missingmethodexception.-ctor?view=netframework-4.8#System_MissingMethodException__ctor_System_String_)

Reply:
From recent experience, it seems that the Docs team prefers to keep simple messages like this one in plain english if possible, without adding crefs.

Reply:
Fair enough, but there's no requirement that it be localized, which is mainly what I was editing out.

Reply:
I've eliminated the cref and simply changed it to "string".

Reply:
Ah I see what you mean, Jeremy, thanks for catching that.